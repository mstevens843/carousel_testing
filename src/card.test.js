// smoke and snapshot tests for cards.js
import { render } from '@testing-library/react';
import Card from './Card'


test('Card component renders without crashing', () => {
    render(<Card caption="Test-image" src="test.jpg" currNum={1} totalNum={3} />);
});


test('Card component matches snapshot', () => {
    const { asFragment } = render(<Card caption="Test Image" src="test.jpg" currNum={1} totalNum={3} />);
    expect(asFragment()).toMatchSnapshot(); 
});

