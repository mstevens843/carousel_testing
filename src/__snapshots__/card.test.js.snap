// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card component matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="Card"
  >
    <h4
      class="Card-title"
    >
      Test Image
    </h4>
    <img
      alt="Test Image"
      class="Card-image"
      src="test.jpg"
    />
    <small
      class="Card-small"
    >
      Image 1 of 3.
    </small>
  </div>
</DocumentFragment>
`;
